backend/routes/quests.go

```diff
@@ -47,10 +45,29 @@ type MainQuest struct {
 	Reward      int    `json:"reward"`
 }
 
+type UserMainQuest struct {
```

Comment:
Can you combine this and the daily one and name it more appropriately for the query changes commented below.

Reply:
not sure i understand what you mean here @b-j-roberts

Reply:
Instead of having 2 separate structs containing the same types and being used for the same thing, it'd be better to just use 1 type. In this example you have`UserMainQuest` and `UserDailyQuest` which are the same

Reply:
i see

Reply:
UserDailyQuest will have type since we are pulling questId and type from DailyQuests, as such they wont have the same type